kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -193,6 +193,13 @@
         required: false,
         default: '',
       },
+      // Alternative to using metaInfo in a top level component to set the
+      // title of the HTML Document
```

Comment:
Well, the ultimate intent is that the metaInfo is not set on another component, but set as a prop on CoreBase - as I suggested in the PR description though, once implemented, I am not sure that implementing this prop buys us that much.

Reply:
Ah I think I understand what you're saying: because it's just as easy to drop `metaInfo` into `SomePage` as it is to pass a `pageTitle` prop into `CoreBase` from `SomePage`?

I agree.

I lean slightly towards a prop because it uses standard Vue mechanisms. `metaInfo` feels a little magical and I wouldn't mind keeping it isolated to a single component.